packages/components/src/components/Dialog/Dialog.stories.tsx

```diff
@@ -291,3 +292,30 @@ export const MobileView: Story = {
         children: 'Hello World',
     },
 };
+
+export const ExternalControl: Story = {
```

Comment:
To match Dropdown and Select story name
```suggestion
export const ControlledComponent: Story = {
```